main.js

```diff
@@ -50,12 +57,22 @@ Apify.main(async () => {
 
     const uniqueKeys = input.uniqueKeys;
     const timestampAttr = input.timestampAttr;
+    
+    if (input.transformFunction) {
+        eval(input.transformFunction);
+        if (!transform || (typeof transform != "function")) {
+            throw new Error('Transform function is not correctly defined! Please consult readme.');
```

Comment:
Duplicate code, can we checked if exists once or put this code in function.

Reply:
It should be correct now.